.devops/full.Dockerfile

```
@@ -6,7 +6,8 @@ RUN apt-get update && \
     apt-get install -y build-essential python3 python3-pip
 
 RUN pip install --upgrade pip setuptools wheel \
-    && pip install numpy requests sentencepiece torch tqdm
+    && pip install numpy requests sentencepiece tqdm \
+    && pip install torch==2.0.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
```

According to the Torch website, this should work to install the cpu-only variant and does not pin the version.

```suggestion
    && pip3 install torch --index-url https://download.pytorch.org/whl/cpu
```

Can you please test if that works as advertised (the latest torch is installed and only with cpu-support)?

---

Worked as advertised - image stays at 1.62GB torch CPU is installed